src/webglimpse/plot/axis.ts

```diff
@@ -83,14 +109,27 @@ export class Axis1D {
     }
 
     pan(vAmount: number) {
+        if (vAmount + this._vMin < this._vMinLimit) {
+            this._vMax = this._vMinLimit + this.vSize;
+            this._vMin = this._vMinLimit;
+            this._limitsChanged.fire();
+            return;
+        } else if (vAmount + this._vMax > this._vMaxLimit) {
+            this._vMin = this._vMaxLimit - this.vSize;
+            this._vMax = this._vMaxLimit;
+            this._limitsChanged.fire();
+            return;
+        }
         this._vMin += vAmount;
         this._vMax += vAmount;
         this._limitsChanged.fire();
     }
```

Comment:
Move last bit into an else to remove duplicate code.

```suggestion
    pan(vAmount: number) {
        if (vAmount + this._vMin < this._vMinLimit) {
            this._vMax = this._vMinLimit + this.vSize;
            this._vMin = this._vMinLimit;
        } else if (vAmount + this._vMax > this._vMaxLimit) {
            this._vMin = this._vMaxLimit - this.vSize;
            this._vMax = this._vMaxLimit;
        } else {
            this._vMin += vAmount;
            this._vMax += vAmount;
        }
        this._limitsChanged.fire();
    }
```